services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -20,6 +27,121 @@ const tags = [
     faker.random.word(),
   ]),
 ];
+const generateRule = (ruleName: string, options: InlineRule, groupedBy: string[] = []) => {
+  return {
+    [ruleName]: {
+      rule: {
+        name: ruleName,
+      },
+      options,
+      groupedBy,
+    },
+  };
+};
+
+const getRuleDefinitions = () => {
+  // Rule ID: PAY_HCA_CC
+  // Description: High Cumulative Amount - Inbound - Customer (Credit Card)
+  // Condition: Sum of incoming transactions over a set period of time is greater than a limit of credit card.
+  const _PAY_HCA_CC = {
+    ruleName: 'PAY_HCA_CC',
+    groupedBy: ['businessId'],
+    options: {
+      groupByBusiness: true,
+      havingAggregate: AggregateType.SUM,
+
+      direction: 'Inbound',
+      excludedCounterpartyIds: ['9999999999999999', '999999******9999'],
+
+      paymentMethods: [PaymentMethod.CreditCard],
+      excludePaymentMethods: false,
+
+      timeAmount: 7,
+      timeUnit: 'days',
+
+      amountThreshold: 1000,
+    } as TransactionsAgainstDynamicRulesType,
+  };
+
+  // Rule ID: PAY_HCA_APM
+  // Description: High Cumulative Amount - Inbound - Customer (APM)
+  // Condition: Sum of incoming transactions over a set period of time is greater than a limit of APM.
+  const _PAY_HCA_APM = {
+    ruleName: 'PAY_HCA_APM',
+    groupedBy: ['businessId'],
+    options: {
+      groupByBusiness: true,
+      havingAggregate: AggregateType.SUM,
+
+      direction: 'Inbound',
+      excludedCounterpartyIds: ['9999999999999999', '999999******9999'],
+
+      paymentMethods: [PaymentMethod.CreditCard],
+      excludePaymentMethods: true,
+
+      timeAmount: 7,
+      timeUnit: 'days',
+
+      amountThreshold: 1000,
+    } as TransactionsAgainstDynamicRulesType,
```

Comment:
All these instances fit the `satisfies` keyword's use-case. 🙂

Reply:
i used it and than i faced with issues from prisma